Gemfile

```diff
@@ -1,11 +1,4 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
+source 'https://rubygems.org'
 
-gem 'pupperware',
-    :git => 'https://github.com/puppetlabs/pupperware.git',
-    :branch => 'main',
-    :glob => 'gem/*.gemspec'
-
-group :test do
-    gem 'rspec'
-    gem 'rspec_junit_formatter'
-end
+gem 'github_changelog_generator'
```

Comment:
We usually pin the gems:
```
group :release do
  gem 'faraday-retry', '~> 2.1', require: false
  gem 'github_changelog_generator', '~> 1.16.4', require: false
end
```
And newer github_changelog_generator have a weak dependency to faraday-retry.

Reply:
thx, will update